src/components/KeyboardShortcuts/index.tsx

```diff
@@ -126,6 +178,14 @@ export class KeyboardShortcutsBase extends React.Component<Props> {
             <kbd>j</kbd>
           </dt>
           <dd>{gettext('Down file')}</dd>
+          <dt className={disabledClassname}>
+            <kbd className={disabledClassname}>p</kbd>
```

Comment:
Since you're already adding a nested style for `kbd` then I say don't bother adding a class name here. Just let the `dt` class name do the styling.